source/includes/api/_fields.md

```diff
@@ -48,51 +44,412 @@ curl -X GET https://api.resourcewatch.org/v1/fields/1170030b-9725-4bfe-8fb4-1b0e
+Once the dataset has been created and its status is set to `saved`, you will be able to use this endpoint to get details about a its fields. The resulting response contains two root fields
 
-> Example
+- `tableName`: a string containing the table name, as stored in the dataset object. The value and function of this field will depend on the dataset's provider, and you can refer to the [dataset](#dataset) documentation for more details about this.
+- `fields`: a key-value pair of field names and their respective details. 
 
-```shell
-curl -i  -XGET 'http://api.resourcewatch.org/v1/fields/491ae6fe-6767-44d1-b5c3-c7b8b384bb7a'
+While the aim of this endpoint is to provide an homogeneous view of data stored in different systems, you will still encounter slight variations in data, depending on the underlying type of the dataset you're querying. The example response on the right illustrates the typical basic response structure: `tableName` with a string value, and a `fields` map that matches each field's name to a `type`. You'll find this structure throughout all responses, but you may also find additional details for each field, and the data types may vary in name. In the next sections we'll cover some of the specifics for each dataset type.
+
+
+### Carto
+
+> Example response for a Carto dataset
+
+```
+{
+    "tableName": "gadm28_adm1",
+    "fields": {
+        "cartodb_id": {
+            "type": "number",
+            "pgtype": "int4"
+        },
+        "the_geom": {
+            "type": "geometry",
+            "wkbtype": "Unknown",
+            "dims": 2,
+            "srid": 4326
+        },
+        "the_geom_webmercator": {
+            "type": "geometry",
+            "wkbtype": "Unknown",
+            "dims": 2,
+            "srid": 3857
+        },
+        "objectid": {
+            "type": "number",
+            "pgtype": "int8"
+        },
+        "iso": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "centroid": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "area_ha": {
+            "type": "number",
+            "pgtype": "float8"
+        },
+        "topojson": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "the_geom_simple": {
+            "type": "geometry",
+            "wkbtype": "Unknown",
+            "dims": 2,
+            "srid": 4326
+        },
+        "geojson": {
+            "type": "string",
+            "pgtype": "text"
+        }
+    }
+}
+```
+
+Carto datasets provide a `type` for each field, as well as additional details, depending on the field type. Most fields will have a `pgtype` field, which will identify a sub-type of sorts - for example, `type` number is matched with different `pgtype` values in this example. This is related with the underling Carto implementation, that uses PostgreSQL, PostGIS and its own software stack. We recommend relying on the `type` as much as possible.
```

Comment:
Is it safe to refer users to [PG docs on data types](https://www.postgresql.org/docs/9.5/datatype.html) (i.e. does this link reflect the list of the values users will find in the `pgtype` field)? If so, we should add this reference here, for completion's sake

Reply:
yes in the end the carto ones are a common postgres + postgis DB types